webpack.config.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+
+const production = process.env.NODE_ENV === 'production';
+
+let config = {
+    context: path.resolve(__dirname, 'webserver', 'static'),
+    entry: {
+        common: ['./scripts/common.js'],
+        datasets: ['./scripts/datasets.js'],
+        global: ['./scripts/global.js'],
+        homepage: ['./scripts/homepage.js'],
+        profile: ['./scripts/profile.js'],
+        stats: ['./scripts/stats.js'],
+        main: ['./styles/main.less']
+    },
+    output: {
+        chunkFilename: production ? '[name].[chunkhash].js' : '[name].js',
+        filename: production ? '[name].[chunkhash].js' : '[name].js',
+        path: path.resolve(__dirname, 'webserver', 'static', 'build'),
+        publicPath: '/static/build/'
+    },
+    mode: production ? 'production' : 'development',
```

Comment:
I *think* this is redundant, since calling webpack with the `-p` flag sets mode to production automatically, and the default is development.